pkg/kudoctl/cmd/env/kudohome/home.go

```diff
@@ -0,0 +1,31 @@
+package kudohome
```

Comment:
here I don't understand why home is in env? env is super generic name - overall I am bit confused around the code structure. Maybe home should be just in /cmd/init/home.go - the same for env.

Reply:
env is something I want to pass around as an initialized structure... it can not live in cmd because it would create a circular dependency issue.